cw-orch-daemon/src/lib.rs

```diff
@@ -23,6 +24,7 @@ pub mod tx_builder;
 pub use self::{builder::*, channel::*, core::*, error::*, state::*, sync::*, tx_resp::*};
 pub use cw_orch_networks::chain_info::*;
 pub use cw_orch_networks::networks;
+pub use env::DaemonEnvVars;
```

Comment:
Do we need to re-export it if it's already exposed under `env`?

Reply:
Yeah, it's mostly used internally so shouldn't need re-export.